scripts/hostcfgd

```diff
@@ -1177,6 +1182,25 @@ class KdumpCfg(object):
                 num_dumps = data.get("num_dumps")
             run_cmd(["sonic-kdump-config", "--num_dumps", num_dumps])
 
+            # Remote option
+            remote = self.kdump_defaults["remote"]
+            if data.get("remote") is not None:
+                remote = data.get("remote")
+            run_cmd(["sonic-kdump-config", "--remote", remote])
+
+            # SSH key
+            ssh_key = self.kdump_defaults["SSH_KEY"]
+            if data.get("SSH_KEY") is not None:
+                ssh_key = data.get("SSH_KEY")
+            run_cmd(["sonic-kdump-config", "--ssh_key", ssh_key])
+
+            # SSH path
+            ssh_path = self.kdump_defaults["SSH_PATH"]
+            if data.get("SSH_PATH") is not None:
+                ssh_path = data.get("SSH_PATH")
+            run_cmd(["sonic-kdump-config", "--ssh_path", ssh_path])
```

Comment:
Why should we execute run_cmd with following defaults when KDUMP remote is not desired by the user?
SSH_KEY": "<user@server>",   # New feature: SSH key, default value
            "SSH_PATH": "<path>"

Reply:
Hi @venkatmahalingam i have update the code as suggested above .